Cython/Utility/NumpyImportArray.c

```diff
@@ -0,0 +1,15 @@
+///////////////////////// NumpyImportArray.init ////////////////////
+
+// comment below is deliberately kept in the generated C file to
+// help users debug where this came from:
+/*
+ * Cython has automatically inserted a call to _import_array since
+ * you didn't include one when your cimported numpy. To disable this
+ * add the line
+ *   <void>_import_array
+ */
+#ifdef NPY_NDARRAYOBJECT_H /* numpy headers have been included */
+#if !NO_IMPORT_ARRAY /* can be explicitly set to tell Numpy its initialization is done manually */
+if (_import_array() == -1) __PYX_ERR(1, 1, __pyx_L1_error) /* __pyx_L1_error reliably exists in __Pyx_InitGlobals */
```

Comment:
> `__PYX_ERR(1, 1, __pyx_L1_error)`

We can pass this in from `code.error_goto(…)` and use Tempita utility code generation for it. That may seem overhead for such a simple thing, but we avoid knowing the details here that are otherwise hidden behind the code writer everywhere else.

Also, I think this code should do what `import_array()` does, i.e. make sure it raises `ImportError` on failure and nothing else.